build.sbt

```diff
@@ -42,6 +43,10 @@ lazy val core = project
       "org.slf4j"                  %  "slf4j-log4j12"              % slf4jVersion % Test,
       "org.specs2"                 %% "specs2-core"                % specsVersion % Test,
       "org.specs2"                 %% "specs2-scalaz"              % specsVersion % Test,
-      "org.specs2"                 %% "specs2-scalacheck"          % specsVersion % Test
+      "org.specs2"                 %% "specs2-scalacheck"          % specsVersion % Test,
+
+      // mongo's `getDatabase` and `getCollection` have `Nullable` annotations and they raise
+      // warnings w/o this dependency
+      "com.google.code.findbugs"   %  "jsr305"                      % jsrVersion % Optional
```

Comment:
I think this should be `provided` instead of `optional`? (i.e. available at compile time but not runtime)

Reply:
I would just remove the qualifier. There's no reason to use `"optional"` or `"provided"` unless writing a library, which this is not.

Reply:
@djspiewak Oh you mean because it will be available at runtime anyway because it's using the datasource plugin mechanism (which doesn't recognize them)?

Reply:
Actually I just mean there's no reason *not* to make it a mandatory dependency. :-)